code/modules/reagents/chemistry/machinery/smoke_machine.dm

```diff
@@ -36,7 +36,7 @@
 
 /obj/machinery/smoke_machine/Initialize(mapload)
 	. = ..()
-	create_reagents(REAGENTS_BASE_VOLUME)
```

Comment:
Weird but sure?

Reply:
I just went through things that create reagents and added injectable to some things that don't lead to any harm. In case of smoke machine, you could extend it's capacity with a drip, or feed reagents very slowly to make it release smoke with bigger delays.

Reply:
I mean it’s already plumbing compatible and everything, feels a bit arbitrary to randomly make smoke machine injectible. But like sure? It’s very minor